CMakeLists.txt

```diff
@@ -336,11 +353,14 @@ find_package(Boost ${VIAMCPPSDK_BOOST_VERSION_MINIMUM} REQUIRED COMPONENTS heade
 # another try via pkgconfig.
 
 find_package(gRPC ${VIAMCPPSDK_GRPC_VERSION_MINIMUM} CONFIG)
+
+# Protobuf has an idiosyncratic system for checking version compatibility, so we specify
+# no version here and check it manually below.
+find_package(Protobuf CONFIG)
```

Comment:
What do you think about below on line 361 making this `gRPC_FOUND AND Protobuf_FOUND`, and then making line 381 (the `else()` be an `elseif !gRPC_FOUND AND !Protobuf_FOUND`, and then adding an `else` that errors out. I just worry a bit about a situation where we find one via `find_package` and the other via `pkg_check_modules` or something silly and then everything is confusingly broken.

Reply:
Good idea we may as well err on the side of caution